projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -218,7 +248,11 @@ export class AuthService implements OnDestroy {
 
   private shouldHandleCallback(): Observable<boolean> {
     return of(this.window.location.search).pipe(
-      map((search) => search.includes('code=') && search.includes('state='))
+      map(
+        (search) =>
+          (search.includes('code=') || search.includes('error=')) &&
+          search.includes('state=')
+      )
```

Comment:
Refactored this to also check for `error` in the querystring, otherwise the SDK doesn't process it.

Reply:
this addition doesn't seem to be unit-test covered

Reply:
I've added a test now to ensure that processing still takes place when there are `error` and `state` params; apologies for the omission.